profile-widget/profilewidget2.cpp

```diff
@@ -1424,21 +1424,21 @@ void ProfileWidget2::contextMenuEvent(QContextMenuEvent *event)
 	QMenu *changeMode = m.addMenu(tr("Change divemode"));
 	if (divemode != OC) {
 		QAction *action = new QAction(&m);
-		action->setText("OC");
+		action->setText(tr("Open circuit"));
```

Comment:
why wouldn't we use divemode_text[OC] or something like this - otherwise we have the same string for translation multiple times...